### ISSUE TITLE
[FEAT.] : Documentation on how to install/use on Linux

### ISSUE THREAD

**Issue by jessstrap** [Zagrios/bs-manager#468]
## Problem

The readme provides installation instructions and a link to releases for Windows users, but none for Linux users. 

## Solution

Add something to the readme giving directions for how Linux users should download, install, and run BSManager. 
Add something to the readme giving directions for how Linux users should launch a downloaded and modded version of BeatSaber.

## Alternative solutions (if any)

Something in the readme pointing Linux users to issues, pull requests, or other projects. 

## Additional context

I got some progress by running the (Windows) installer in steam as a non steam app and forcing a specific version of proton but it quickly became clear I was up the wrong tree. After the install I went digging in the file tree to find the installed BSManager.exe to add as the next stage of non steam app. Then during the first run when I tried to download a version of the game it told me to install .net. That sent me to a download page for the installer for the Windows version of .net.

**Comment by jessstrap** [comment]
@Insprill I think this is up your alley. Alternatively If you could give me the top level instructions I can do them and make a pull request for the readme myself.

**Comment by Insprill** [comment]
I've made an AUR package for it that works out-of-the-box, but I haven't looked into what's required for other distros/packaging formats.

It's also not fully functional on Linux yet, with the main feature lacking being the ability to actually start the game. I have an open PR fixing this, but there's still some issues in-game related to how it's launched that aren't present when launching through Steam.

I think the documentation would be better off waiting until it's fully functional since things might change in the meantime.

**Comment by jessstrap** [comment]
There's an AUR package? Excellent. That gives me something to work from for my own purposes. I'm a brand new baby arch user after being on XUbuntu for 15ish years. Back to the readme...

How about this, I make a new issue 'Linux support meta issue' with a link to your AUR package, then update the readme with a note that Linux support is WIP and a link to that issue?

**Comment by kevin-wijnen** [comment]
Hi folks :wave: 

As [@silentrald's comment suggests the lack of Linux documentation](https://github.com/Zagrios/bs-manager/issues/583#issuecomment-2365197824) in my issue #583, I figured helping out with how I got it to work on my EndeavourOS (Arch Linux distro) system.

---

I've used the **AppImage** build, starting with **1.5.0-alpha4**. This simply works under X11 and Wayland sessions on the desktop. It does **not** launch nor render it properly on SteamVR (via a *non-Steam game shortcut*, createable in Steam).

To get BSManager's AppImage to launch on my system, I had to do the following:

- Make sure screen and the dotnet packages are installed (as per #578, although this may become redundant once #586 is merged)

To get BSManager's AppImage to launch via SteamVR, I had to do the following:

- Create a non-Steam game shortcut by Steam
    - Click on the bottom left corner "Add game" button on the Steam client, then the first option.
- Edit the shortcut's launch options to include `LD_PRELOAD= %command%`
    - Steam by default overrides libraries for the Steam runtime. You can prevent that from happening by using the `LD_PRELOAD` environment variable pointing to nothing, to let the app use the system's libraries.

Currently, this allows the AppImage to run via SteamVR, but it does not get rendered properly. As discussed in https://github.com/Zagrios/bs-manager/issues/583#issuecomment-2365402251, a third party overlay (such as [wlx-overlay-s](https://github.com/galister/wlx-overlay-s) or Valve's [Gamescope, once modified to run through VR](https://www.s1sw.xyz/gamescope-tutorial.html))

Specific to Beat Saber on Linux, I noticed a weird jittering when moving my head. That does not happen on other SteamVR games. My found solution is to turn on Legacy Reprojection, in SteamVR's game specific video settings:

![image](https://github.com/user-attachments/assets/bc7318d0-207e-4ea0-8783-48dbb4fb6e57)

---

I think it would be better to get @GaetanGrd involved, as they are the designated Documentation Lead (according to [the project's README](https://github.com/Zagrios/bs-manager?tab=readme-ov-file#credits)).
